src/builder/context.rs

```diff
@@ -19,6 +19,9 @@ use super::timer;
 use file_util::create_dir;
 use process_util::PROXY_ENV_VARS;
 
+pub const PROTECTED_DIRS: &'static [&'static str] = &[
+    "dev", "proc", "run", "sys", "tmp", "work",
```

Comment:
Thought about this but we don't know how somebody will use such containers. So I decided that several empty directories are not a big deal.